frontend/src/components/Explore.tsx

```diff
@@ -34,13 +34,16 @@ export const Explore = (): JSX.Element => {
 	const pageMetadata = useSelector(
 		(state: RootState) => state.dataset.datasets.metadata
 	);
+	const pageMyDatasetsMetadata = useSelector(
+		(state: RootState) => state.dataset.myDatasets.metadata
```

Comment:
I don't see that you added myDatasets under RootState.

It doesn't work as it find myDatasets as defined. Declare it under RootState in data.ts

Reply:
I added the state and new actions and methods. Should be working now.